nc_files/remap-subroutines/qt_auto_probe_tool.ngc

```diff
@@ -61,22 +66,27 @@ O400 endif
 
 F #<_hal[qtversaprobe.searchvel]>
 G91
-G38.2 Z- #<_ini[VERSA_TOOLSETTER]MAXPROBE>
+G38.3 Z- #<_ini[VERSA_TOOLSETTER]MAXPROBE>
 G0 Z #<_hal[qtversaprobe.backoffdist]>
 
-;reprobe at probe speed
-F #<_hal[qtversaprobe.probevel]>
-G38.2 Z- [#<_hal[qtversaprobe.backoffdist]> *1.2]
-
 O500 if [#5070 EQ 0]
 G90
 O500 return [-3] ; indicate probe contact failure to epilog
 O500 endif
 
+;reprobe at probe speed
+F #<_hal[qtversaprobe.probevel]>
+G38.3 Z- [#<_hal[qtversaprobe.backoffdist]> *1.2]
+
+O600 if [#5070 EQ 0]
+G90
+O600 return [-3] ; indicate probe contact failure to epilog
+O600 endif
+
 G90
 G53 G0 Z[#<_ini[CHANGE_POSITION]Z>]
 
-#<touch_result> = #5063
+#<touch_result> = #5063 + #<zworkoffset>
```

Comment:
Need brackets around the addition:
#<touch_result> = [#5063 + #<zworkoffset>]

Reply:
Done.